src/wdio/services/TerraCommands/hide-input-caret.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Hides the blinking input caret that appears in editable text areas and can cause inconsistent test failues.
```

Comment:
nit: This reads a little strange. I think the `and` makes it sound like this helper method hides the caret and can cause inconsistent test failures.

It might be worth rephrasing it slightly. Perhaps something like along these lines:

Hides the blinking input caret that appears within editable text areas to prevent inconsistent test failures.